osu.Game/Skinning/LegacySkin.cs

```diff
@@ -385,26 +356,15 @@ public override Drawable GetDrawableComponent(ISkinComponent component)
                                 }
                             })
                             {
-                                Children = this.HasFont(LegacyFont.Score)
-                                    ? new Drawable[]
-                                    {
-                                        new LegacyComboCounter(),
-                                        new LegacyScoreCounter(),
-                                        new LegacyAccuracyCounter(),
-                                        new LegacyHealthDisplay(),
-                                        new SongProgress(),
-                                        new BarHitErrorMeter(),
-                                    }
-                                    : new Drawable[]
-                                    {
-                                        // TODO: these should fallback to using osu!classic skin textures, rather than doing this.
-                                        new DefaultComboCounter(),
-                                        new DefaultScoreCounter(),
-                                        new DefaultAccuracyCounter(),
-                                        new DefaultHealthDisplay(),
-                                        new SongProgress(),
-                                        new BarHitErrorMeter(),
-                                    }
+                                Children = new Drawable[]
+                                {
+                                    new LegacyComboCounter(),
+                                    new LegacyScoreCounter(),
+                                    new LegacyAccuracyCounter(),
+                                    new LegacyHealthDisplay(),
+                                    new SongProgress(),
+                                    new BarHitErrorMeter(),
+                                }
```

Comment:
What's with these changes? Is it because the fallback is now being done?

Reply:
Yep, check commit message on https://github.com/ppy/osu/pull/17416/commits/a4d17a915f96c505e518dd261f9b4d1d5d967b7f for in depth explanation (wasn't in opening post because it was found in @bdach  review pass)